qa/tasks/ceph_test_case.py

```diff
@@ -38,6 +38,7 @@ class CephTestCase(unittest.TestCase):
 
     def setUp(self):
         self._mon_configs_set = set()
+        self.config_set('mds', 'defer_client_eviction_on_laggy_osds', 'false')
```

Comment:
@dparmar18 @batrick while this does fix/suppress the api test failure, i am more curious on how this newly added health check `MDS_CLIENTS_LAGGY` will clear itself. Is this error always going to be present in the test environment? And also how's an OSD considered laggy?

Reply:
@batrick added some explanation here: https://tracker.ceph.com/issues/61907#note-9

Reply:
Let's move this to `qa/tasks/mgr/mgr_test_case.py` (sp?) instead.